docs/js/map_old_links.js

```diff
@@ -33,13 +33,13 @@ document.addEventListener("DOMContentLoaded", function (event) {
       prepping: "#section/Image-Prepping",
       standard_fonts: "#section/Standard-PDF-Fonts",
       addresses: "#tag/Addresses",
-      addresses_object: "#operation/get_address",
-      addresses_create: "#operation/create_address",
-      addresses_retrieve: "#operation/get_address",
-      addresses_list: "#operation/list_addresses",
+      addresses_object: "#operation/address_retrieve",
+      addresses_create: "#operation/address_create",
+      addresses_retrieve: "#operation/address_retrieve",
+      addresses_list: "#operation/addresses_list",
       us_verifications: "#tag/US-Verifications",
-      us_verifications_object: "#operation/bulk_us_verifications",
-      us_verifications_create: "#operation/bulk_us_verifications",
+      us_verifications_object: "#operation/us_verifications_bulk",
```

Comment:
why `us_verifications_bulk` and not `verifications_us_bulk`?

I'm an advocate of the BEM naming style (Block Element Modifier). The term doesn't 100% correlate with BE engineering but the concept does